Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Templates/ExecutableAssembly.tt

```diff
@@ -0,0 +1,56 @@
+﻿<#@ template language="C#" #>
+<#@ assembly name="System.Core" #>
+<#@ import namespace="System.Linq" #>
+<#@ import namespace="System.Text" #>
+<#@ import namespace="System.Collections.Generic" #>
+<#@ import namespace="System.Threading.Tasks" #>
+<#@ import namespace="Amazon.Lambda.Annotations.SourceGenerator.Extensions" #>
+<#@ import namespace="Amazon.Lambda.Annotations.SourceGenerator.Validation" #>
+<#@ import namespace="Amazon.Lambda.Annotations.SourceGenerator.Models" #>
+<#@ import namespace="Amazon.Lambda.Annotations.SourceGenerator.Models.Attributes" #>
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Amazon.Lambda.Core;
+
+namespace <#= this._containingNamespace #>;
+
+public class GeneratedProgram
+{
+    private static async Task Main(string[] args)
+    {
+
+        switch (Environment.GetEnvironmentVariable("ANNOTATIONS_HANDLER"))
+        {
+<#
+        foreach (var model in this._lambdaFunctions)
+        {
+            #>
+            case "<#=model.LambdaMethod.Name#>":
+<#
+            if (model.GeneratedMethod.ReturnType.FullName == "void")
+            {
+#>
+                Action<<#= string.Join(", ", model.GeneratedMethod.Parameters.Select(p => $"{p.Type.FullName}")) #>> <#= model.LambdaMethod.ExecutableAssemblyHandlerName #> = new <#= model.LambdaMethod.ContainingNamespace #>.<#= model.LambdaMethod.ContainingType.Name #>_<#= model.LambdaMethod.Name #>_Generated().<#= model.LambdaMethod.Name #>;
```

Comment:
If there are no parameters like the following function

```csharp
    [LambdaFunction]
    public void Post()
    {

    }
```

then the Action is setup with an empty generic parameter instead of no generic parameter
```
            case "Post":
                Action<> post_handler = new JamesAnnotationTest.Functions_Post_Generated().Post;
                await Amazon.Lambda.RuntimeSupport.LambdaBootstrapBuilder.Create(post_handler, new Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer()).Build().RunAsync();
                break;
```